content/aips/7cffae5fb3c0-AIP-Add_LUSD_to_Aave_V2.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
I think this file is unnecessary, as it only contains the descriptions to fill. And you have the other file filled

Reply:
Yes, right. I initially created this one (with the output from the hash generation script) and then realized that the rest of them were all uppercase. I forgot to remove it I guess, will do right away, thanks!